flow/designs/nangate45/ibex/config.mk

```diff
@@ -2,48 +2,32 @@ export DESIGN_NICKNAME = ibex
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_register_file_ff.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_register_file_fpga.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_register_file_latch.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_wb_stage.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_badbit_ram_1p.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_clock_gating.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_generic_clock_gating.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_generic_ram_1p.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_lfsr.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_ram_1p.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_28_22_dec.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_28_22_enc.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_39_32_dec.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_39_32_enc.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_72_64_dec.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_secded_72_64_enc.v \
-                       $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/prim_xilinx_clock_gating.v
-
-
-
+export VERILOG_FILES = $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_pkg.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_alu.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_compressed_decoder.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_controller.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_cs_registers.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_counter.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_decoder.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_ex_block.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_id_stage.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_if_stage.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_wb_stage.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_load_store_unit.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_multdiv_slow.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_multdiv_fast.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_prefetch_buffer.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_fetch_fifo.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_pmp.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_csr.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_core.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_register_file_ff.sv \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/syn/rtl/prim_clock_gating.v
+
+export VERILOG_INCLUDE_DIRS = \
+    $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/vendor/lowrisc_ip/prim/rtl/
+
+export USE_YOSYS_SLANG = 1
```

Comment:
I've converted nangate45/ibex to use this option. I've kept Ibex at the same version (from Nov 2020) but replaced the pre-processed Verilog sources with the SystemVerilog originals